can_tools/scrapers/official/federal/HHS/hhs_state.py

```diff
@@ -108,25 +106,18 @@ def normalize(self, data: str) -> pd.DataFrame:
         }
 
         # Put into long form
-        out = df.melt(id_vars=["dt", "location"], value_vars=crename.keys())
+        out: pd.DataFrame = df.melt(
+            id_vars=["dt", "location"], value_vars=crename.keys()
+        )
         out.loc[:, "value"] = pd.to_numeric(
             out["value"].astype(str).str.replace(",", "").replace("nan", None)
         )
+        # Dropping missing values. As of 2023-06-11 all missing values are from
```

Comment:
Scraper was failing tests b/c there were some NaN values from 2020. Not entirely sure what changed to cause this failure, but this fixes the scraper